Infrastructure.md

```diff
@@ -0,0 +1,58 @@
+# Infrastructure Setup
+
+The provisioning system depends on a central master, hosted at http://classroom.puppet.com
```

Comment:
So, devil's advocate. What do we gain by doing this instead of the bootstrap script just cloning the repo, installing Puppet/PE, and doing puppet apply? That would let us avoid maintaining a PE master. We could host the bootstrap.sh in S3 so the curlbash would still work.